### ISSUE TITLE
Window has 1px border on top

### ISSUE THREAD

**Issue by vainamov** [microsoft/vscode#106801]
When VS Code is closed in maximized state and reopened again the title bar isn't stretched to the top of the screen but instead a white line is rendered.
![grafik](https://user-images.githubusercontent.com/11841072/93233398-4fcb8f00-f77b-11ea-8a55-287e5600c8ff.png)

When manually snapping the window to the top (or fully maximizing it again), a 1px border at the top of the window appears that prevents clicking the close button when moving the mouse to the upper right corner.

I do have a 4K monitor and use windows with 150% scaling, however a few Insider versions back this used to work just fine.

Version: 1.50.0-insider (system setup)
Commit: e13875b77c89b95f20ccb5667e14ff164c198e57
Date: 2020-09-15T05:35:57.362Z
Electron: 9.3.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.19041

Steps to Reproduce:

1. Start VS Code and maximize the window
2. Close VS Code
3. Start VS Code again

Does this issue occur when all extensions are disabled?: Yes

**Comment by IllusionMH** [comment]
Problem with Close button looks like duplicate of #90466 (same for menu items).
Do you have custom `window.zoom` value specified in `settings.json`?
Also do I understand correctly that you have this border not only at the top but also on the sides?


I also just noticed white bar at the top (as shown above) only after application start but right click on it in title area (between menu and minimize) will be registered on VS Code window so it works fine.
Line is gone if I restore and maximize window, but still can't click buttons (can click buttons only when Windows scaling is 100%).
No border on right/left screen edges, clicks works fine there.

SP3 2160x1440 (
Windows scaling 150%
`"window.zoomLevel": 1` (also reproducible without this setting)

Noticed this in:
Version: 1.50.0-insider (user setup)
Commit: cca20eba8974b1d6b6a9b71bf4e60a29c8f26648
Date: 2020-09-12T01:04:21.398Z
Electron: 9.3.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.20211

@deepak1556 is it possible that fix for #85310 results in this border on application start?

**Comment by vainamov** [comment]
Yes, the white line is part of the application and registers clicks for the title bar. My `window.zoom` setting is set to 0, but setting it to 1 or removing it entirely makes no difference. Also I haven't touched this setting since when I first installed VS code several years ago.

Regarding the border I quickly checked to make sure. Turns out it's also only present on the top of the window. I tried to capture this on image, it's hard to see but there is a 1 pixel high dark line on the top, and once I move the cursor (dark line on the right) below that line the hover event is passed through again.
![grafik](https://user-images.githubusercontent.com/11841072/93271604-fb90d100-f7b3-11ea-9f2c-cadde136a6ae.png)

**Comment by lllopo** [comment]
I can confirm to some extent. I'm using a FullHD display with 125% scaling and the white line appears as stated in the original issue. But when I restore the window and maximize it again - it disappears and everything looks normal instead of what happens to the original issue creator. 
I also can confirm it appeared first with the initial 1.5.0 build and remains since for the (currently) two consecutive builds.

**Comment by lllopo** [comment]
The issue got fixed for me with the latest insiders :

Version: 1.50.0-insider (user setup)
Commit: 9de65ab77e587cf7ab5dd00ea301cd5431b0f397
Date: 2020-09-29T07:18:01.270Z
Electron: 9.2.1
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.19041

**Comment by vainamov** [comment]
Can confirm, 9de65ab77e587cf7ab5dd00ea301cd5431b0f397 fixed it for me as well.

**Comment by IllusionMH** [comment]
Looks like this was initially caused by Electron 9.3.0 update which was reverted.
Most likely will reoccur after Electron update.

**Comment by lllopo** [comment]
> Looks like this was initially caused by Electron 9.3.0 update which was reverted.
> Most likely will reoccur after Electron update.

Let's hope they'll fix it in the meanwhile :-)

**Comment by deepak1556** [comment]
Merging this to https://github.com/microsoft/vscode/issues/85655
